pkgs/development/python-modules/pytest-watch/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, pkgs
+, pythonPackages
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-watch";
+  version = "4.2.0";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06136f03d5b361718b8d0d234042f7b2f203910d8568f63df2f866b547b3d4b9";
+  };
+
+  doCheck = true;
```

Comment:
```suggestion
  # No Tests
  doCheck = false;
```

Reply:
This just added a comment, there's no tests so the `checkPhase` should be disabled.
Often isn't apparent why tests are disabled so adding a comment that there's just no tests is helpful.

Reply:
Done. I'm going to push the changes as you requested, but I can't build locally without `with pythonPackages;`:
```bash
 $ nix-build -A pythonPackages.pytest-watch                                             
error: undefined variable 'pytest' at /home/dmvianna/sandbox/nixes/nixpkgs/pkgs/development/python-modules/pytest-watch/default.nix:20:19
```